back/app/services/permissions/base_permissions_service.rb

```diff
@@ -1,6 +1,7 @@
 module Permissions
   class BasePermissionsService
     SUPPORTED_ACTIONS = %w[
+      visiting
```

Comment:
This was necessary to turn some specs green during the restructuring work. James confirmed that this makes sense. Basically, `supported_action?` corresponds with whether or not we support granular permissions for these actions.